code/modules/modular_computers/file_system/programs/ntmessenger.dm

```diff
@@ -294,8 +294,11 @@
 
 	if (!string_targets.len)
 		return FALSE
-
-	if (prob(1))
+	var/sent_prob = 1
+	if(ishuman(user))
+		var/mob/living/carbon/human/old_person = user
+		sent_prob = old_person.age >= 30 ? 25 : 1
```

Comment:
```suggestion
	var/sent_prob = 1
	if(ishuman(user))
		var/mob/living/carbon/human/old_person = user
		sent_prob = old_person.age >= 30 ? 25 : sent_prob
```

less places to touch in case someone wants to tweak it in the future